### ISSUE TITLE
could not connect to dbus session bus

### ISSUE THREAD

**Issue by lilmike** [mate-desktop/mate-session-manager#128]
Hi,
I just updated mate to 1.16.0 on arch Linux, and when starting nodm, mate gives me a dialogue that says 'Could not connect to session bus at /tmp/dbus-xxxxx' where xxxxx is a random string of numbers and letters. Unless mate is doing something I'm totally not aware of (which could be the case) my session bus is at /run/user/1000/bus. Any idea why it might be doing this?

Thanks,
-Michael.

**Comment by monsta** [comment]
I don't think we changed anything related to that. Just in case, please make sure you have the whole MATE upgraded to 1.16, and also try other display managers like LightDM.

**Comment by lilmike** [comment]
Hi,
I tried slim, it seemed to have the same issue. I also tried startx by itself, and that *did* open mate, but whenever I open anything that needs the bus it claims it's connecting to /tmp/dbus-xxxx also, so for some reason startx is actually opening the bus while nodm and slim aren't, and mate is still opening/trying to open the bus when it shouldn't.
Any idea?
-Michael.

**Comment by monsta** [comment]
Can you also try LightDM just in case? I've never seen this issue with it.
